### PR TITLE
Add blogpost on railway traffic

### PR DESCRIPTION
**Pull request by szarnyasg**
This blog post could be used to demonstrate some common patterns in DuckDB SQL, such as CTAS (starting from CSV, including the CSV shorthand (no read_csv or COPY function required), the FROM-first syntax, and HTTPS support) and UNPIVOT + COLUMNS + EXCLUDE.

### CODE REVIEW

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+## Introduction
+
+The Netherlands, the birthplace of DuckDB, is a small country with an area of about 42,000&nbsp;km².
+It has a [dense railway network](https://en.wikipedia.org/wiki/Rail_transport_in_the_Netherlands), which consists of 3,223 km of tracks and 397 stations.
+
+Information about this network, including its schedule, distances and stations, is available as an [open data set](https://www.rijdendetreinen.nl/en/open-data/).
+This high-quality data set is maintained by the team behind the [Rijden de Treinen _(Are the trains running?)_ application](https://www.ijdendetreinen.nl/en/about).
+
+In this post, we'll demonstrate some of DuckDB's analytical features on the Dutch railway network data set.
+Unlike most of our other blog posts, this one does introduce a new feature or release: instead, it demonstrates several existing features using a single domain.
```
Comment (Tishj) [inline _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:17]: does -> doesn't

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+## Loading the Data
+
+<!--
+    core features used: CSV reader, CTAS, aggregration
+    demonstrated friendly features: FROM-first syntax, format pretty-printer
+    extensions used: -
+-->
+
+For our initial queries, we'll use the 2023 railway schedule.
+To get this data set, download the [`services-2023.csv.gz` file](https://blobs.duckdb.org/data/nl-railway/services-2023.csv.gz) (330 MB) and load it to DuckDB.
```
Comment (Tishj) [inline _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:29]: load it to -> load it into

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+
+<!--
+    core features used: GROUP BY ALL
+    extensions used: -
+-->
+
+Let's ask a simple query first: _What were the busiest railway stations in the Netherlands in the first 6 months of 2023?_
+
+First, let's compute the number of services passing through each station for every month.
+To do so, we extract the month from the service's date, then performs a group-by aggregation with a `count(*)`:
```
Comment (Tishj) [inline _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:79]: performs -> perform

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+
+It returns the following result:
+
+| Service:Date | Stop:Station name  |
+|--------------|--------------------|
+| 2023-01-01   | Rotterdam Centraal |
+| 2023-01-01   | Delft              |
+| 2023-01-01   | Den Haag HS        |
+
+> For querying private S3 buckets, you can use DuckDB's [Secrets Manager](/docs/extensions/httpfs/s3api#configuration-and-authentication) to authenticate.
```
Comment (Tishj) [inline _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:187]: Do we want to include this?
We have another blog post covering the secret manager and it's not used here, I feel it's a little out of place here

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+
+|     station1     |      station2      | distance |
+|------------------|--------------------|---------:|
+| Eemshaven        | Vlissingen         | 426      |
+| Eemshaven        | Vlissingen Souburg | 425      |
+| Bad Nieuweschans | Vlissingen         | 425      |
+
+## Conclusion
+
+In this post, we demonstrated some of DuckDB's key feaures,
```
Comment (Tishj) [inline _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:404]: `feaures` -> `features`

File: _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md (hunk 2 above)
Comment (szarnyasg) [reply _posts/2024-05-31-analyzing-railway-traffic-in-the-netherlands.md:29]: fixed

**Pull request merged**